hoomd/mpcd/geometry.py

```diff
@@ -124,6 +124,78 @@ def _attach_hook(self):
         super()._attach_hook()
 
 
+class RotatedParallelPlates(Geometry):
+    r"""Rotated parallel-plate channel.
+
+    Args:
+        separation (float): Distance between plates.
+        angle (float): Rotation angle in degrees.
+        speed (float): Wall speed.
+        no_slip (bool): If True, surfaces have no-slip boundary condition.
+            Otherwise, they have the slip boundary condition.
+
+    `RotatedParallelPlates` works similarly to parallel plates but angle rotates
+    the plates. The particles are confined between two infinite rotated parallel
+    plates centered around the origin. The plates are placed at
+    :math:`y=-H + x * tan(angle)` and :math:`y=+H + x* tan(angle)`,
+    where the total`separation` is :math:`2H * cos(angle)`.
```

Comment:
A few things:
* I put the single ticks around ParallelPlates so that sphinx will generate a link to it.
* I put the double ticks around angle so it shows up as code
* I recommend trying to write documentation in the way that makes sense for how the user interacts with it, and not necessarily how it's implemented. So although the math you wrote is correct, it is probably not needed to understand how the geometry works! How about something like this instead?

```suggestion
    `RotatedParallelPlates` are similar to `ParallelPlates`, but the plates are
    rotated by ``angle`` relative to the *x* axis. The ``separation`` between
    the plates is constant and has the same meaning as in `ParallelPlates`.
    Setting ``angle = 0` is equivalent to `ParallelPlates`.
```